app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+
+const { getHealthCheck, getTopics } = require("./controllers/controller");
+
+const app = express();
+
+// app.use(express.json());

```

Comment:
Make sure you remove any commented-out code. Anything that you are not currently using you can delete 🔍